rfc/text/basic/bp_sessions.md

```diff
@@ -139,13 +143,19 @@ No response to an `ABORT` message is expected.
         |Client|          |Router|
         `------'          `------'
 
-
 *Example*
 
 {align="left"}
         [3, {"message": "The realm does not exist."},
             "wamp.error.no_such_realm"]
 
+* After session is established, when protocol violation happens (see "Protocol errors" section).
+
+*Example*
+
+{align="left"}
+        [3, {"message": "Received WELCOME message after session was established."},
```

Comment:
I bet you rather meant to write _"Received *HELLO* message after session was established"_

Reply:
Well,  for router it is HELLO, for client peer — it is WELCOME :)

Reply:
Yes ... you're right ... it's a good example no need to change it ;-)